DependencyInjection/Configuration.php

```diff
@@ -938,25 +939,27 @@ public function getConfigTreeBuilder()
                         ->ifTrue(function ($v) { return isset($v['debug']); })
                         ->thenInvalid('The "debug" name cannot be used as it is reserved for the handler of the profiler')
                     ->end()
-                    ->example(array(
-                        'syslog' => array(
+                    ->example(
+                        [
```

Comment:
the opening tag should be kept in its previous location (the new place where you added it would involve increasing the indentation of the content to respect PSR-2 btw)

Reply:
Fixed :+1: